### PR TITLE
adding solution to task 7 post/comment

### PR DESCRIPTION
**Pull request by Topecoding206**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -126,15 +126,60 @@ describe("GET  /api/articles/:article_id/comments", () => {
...
-      .get("/api/articles/nonsense/comments")
+      .get("/api/articles/nonsense/comment")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:145]: What would happen if the user tried to post a comment with an empty body? Implement this in your code

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comment

File: __tests__/app.test.js (hunk 1 above)
Comment (Topecoding206) [reply __tests__/app.test.js:145]: done

Comment (Topecoding206) [reply __tests__/app.test.js:145]: i also implement  test for if only username is post

**Pull request merged**